src/format/parsed.rs

```diff
@@ -627,7 +627,8 @@ impl Parsed {
     /// plus a time zone offset.
     /// Either way those fields have to be consistent to each other.
     pub fn to_datetime(&self) -> ParseResult<DateTime<FixedOffset>> {
-        let offset = self.offset.ok_or(NOT_ENOUGH)?;
+        // If there is no explicit offset, consider a timestamp value as indication of a UTC value.
+        let offset = self.offset.or(self.timestamp.map(|_| 0)).ok_or(NOT_ENOUGH)?;
```

Comment:
I don't think this formulation is very clear. What about something more explicit?

```rust
let offset = match self.timestamp {
    Some(_) => 0,
    None => self.offset.ok_or(NOT_ENOUGH)?,
};
```

Reply:
Agreed that is better to have clear code than a comment.

The order is different though:
```rust
let offset = match self.offset {
    Some(off) => off,
    None => match self.timestamp {
        Some(_) => 0,
        None => return Err(NOT_ENOUGH),
    }
};
```

Do you want to make it this explicit?

Reply:
So is that actually correct? By the reasoning of the original issue, timestamps are supposed to always be in UTC, right?

Reply:
If some input contains both a timestamp and a nonzero offset, I see no reason to start calling that invalid.
I imagine not everyone takes the UTC part of timestamps that seriously.

Reply:
Okay, then I would write it like this:

```rust
let offset = match (self.offset, self.timestamp) {
    (Some(off), _) => off,
    (None, Some(_)) => 0, // UNIX timestamp may assume 0 offset
    (None, None) => return Err(NOT_ENOUGH),
};
```

Reply:
That is better, sorry I didn't come up with it.